src/core/linter.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
```

Comment:
rename this file to executor.js or something that is more general.

Reply:
or remove it. this class doesn't seem so useful (merge with src/core/moenda.js ).